vimrc

```diff
@@ -158,3 +158,6 @@ set diffopt+=vertical
 if filereadable($HOME . "/.vimrc.local")
   source ~/.vimrc.local
 endif
+
+set secure
```

Comment:
I like that a lot. Should we use Git to check further up the tree so that the settings work if you're not at the root folder, or is that getting too cute with configuration magic?